t/08-autotest.t

```diff
@@ -422,6 +424,30 @@ subtest rollback_activated_consoles => sub {
     is_deeply(\@selected_consoles, [{cmd => 'backend_select_console', testapi_console => 'last_milestone_console'}], 'last milestone console selected');
 };
 
+subtest find_script => sub {
+    my $override_path = 0;
+    $bmwqemu::vars{WHEELS_DIR} = '';    # overwrite WHEELS_DIR and set it empty
+    $bmwqemu::vars{ASSETDIR} = Cwd::getcwd . '/t/data/assets';
```

Comment:
Why not use the start.pm fake data? `t/fake/tests/start.pm`

Reply:
This condition is actually covering these lines: https://github.com/os-autoinst/os-autoinst/blob/master/autotest.pm#L56-L58
To cover `$script_override_path‎`